providers/implementations/kdfs/x942kdf.c

```diff
@@ -165,6 +169,9 @@ static int der_encode_sharedinfo(WPACKET *pkt, unsigned char *buf, size_t buflen
  * suppPubInfo (in bits). It is ignored if the value is 0.
  * |cek_oid| The oid of the key wrapping algorithm.
  * |cek_oidlen| The length (in bytes) of the key wrapping algorithm oid,
+ * |acvp| is the optional blob of DER data representing all the fields related
+ * to partyUInfo, partyVInfo, suppPubInfo and suppPrivInfo. It can be NULL.
+ * |acvp_len| is the |acvp| length (in bytes).
```

Comment:
I would amend this comment to have it make it clear that when acvp is non-NULL, the four *blah*Info MUST be NULL.

Reply:
I left it out because the caller was responsible for the check.. Will add anyway.